.github/workflows/test.yml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    environment: test-workflow
```

Comment:
I'm not too familiar with these GH YAML configs. but do you know if this line is necessary / what it signifies or denotes?

Reply:
Yes. That line does loads in the Github test-workflow environment so that the secrets in it are accessible during lines 40-45.

Reply:
Ahh great :) Thanks